app/src/main/res/layout/sample.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/operando/AndroidDangerSample/blob/cedbf8108b88efb0f5a4e5f82a9b022bdb279d7e/app/src/main/res/layout/sample.xml#L2"></span>The resource <code>R.layout.sample</code> appears to be unused</td>
    </tr>
  </tbody>
</table>